article/sections/_9_conclusion.tex

```diff
@@ -40,3 +40,15 @@ \section{Conclusion}
 We then show that while the deconfounder algorithm might be a promising step in the right direction, 
 it still has some notable deficiencies. More specifically, we show that the deconfounder algorithm shows 
 to be sensitive to small errors in one's data.
+
+We hope that this chapter has provided enough preliminary evidence to demand modelers on the importance of using DAGs in their analyses.
```

Comment:
```suggestion
% Causal diagram substituted for DAG because cyclic / undirected causal graphs are sometimes necessary!
% "Overall" added to signal a wrapping up of the section.
Overall, we hope that this chapter has provided enough preliminary evidence to convince demand modelers of the importance of using causal diagrams in their analyses.
```

Reply:
See #174